src/launch/launch_config.cpp

```diff
@@ -127,6 +127,17 @@ bool ParseContext::shouldSkip(TiXmlElement* e)
 	return false;
 }
 
+
+bool ParseContext::checkEmptyString(std::string str)
```

Comment:
can we have this as a static function at the top of the cpp file instead of a member function? It has nothing to do with `ParseContext`.

Reply:
also, please rename to `stringIsEmpty()` to make the polarity clear (empty => true)

Reply:
and use a const reference to avoid copying the string: `const std::string& str`